client/webserver/site/src/js/wallets.js

```diff
@@ -114,7 +114,7 @@ export default class WalletsPage extends BasePage {
     // Clicking on the available amount on the withdraw form populates the
     // amount field.
     bind(page.withdrawAvail, 'click', () => {
-      page.withdrawAmt.value = page.withdrawAvail.textContent
+      page.withdrawAmt.value = page.withdrawAvail.getAttribute('data-amount')
```

Comment:
Might as well grab it directly from the wallet.

```Javascript
const assetID = parseInt(page.withdrawForm.dataset.assetID)
const asset = app().assets[assetID]
page.withdrawAmt.value = asset.wallet.balance.available / asset.info.unitinfo.conventional.conversionFactor
```

Which could be simplified further by persisting the wallet parsed in `this.showWithdraw` as a `WalletsPage` property in a fashion similar to `openAsset`, `reconfigAsset`, etc.